_src/pl/static/sass/_views.scss

```diff
@@ -30,3 +30,18 @@
 .pl.view-element {
 
 }
+
+// ------------------------------
+// #TAB PANELS / EXAMPLES
+// ------------------------------
+.example-set {
```

Comment:
I think this stuff is better defined in the ``_components.scss`` partial in the same directory. Its less of a view and more components on PL doc site specific views.